collector.go

```diff
@@ -153,14 +155,44 @@ var (
 	)
 )
 
+func freeipmiConfig(driver, user, password string) string {
+	return fmt.Sprintf(`
+driver-type %s
+privilege-level admin
+username %s
+password %s
+	`, driver, user, password)
```

Comment:
Is the trailing tab in the config file on purpose?

Reply:
Yes. There is a leading newline as well, i preferred the code readability.